operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -33,14 +34,50 @@ pub async fn raster_stream_to_geotiff_bytes<T, C: QueryContext + 'static>(
 where
     T: Pixel + GdalType,
 {
-    let file_name = format!("/vsimem/{}.tiff", uuid::Uuid::new_v4());
+    let file_path = PathBuf::from(format!("/vsimem/{}.tiff", uuid::Uuid::new_v4()));
+
+    raster_stream_to_geotiff(
+        &file_path,
+        processor,
+        query_rect,
+        query_ctx,
+        no_data_value,
+        spatial_reference,
+        tile_limit,
+    )
+    .await?;
+
+    let bytes = gdal::vsi::get_vsi_mem_file_bytes_owned(
+        file_path
+            .to_str()
+            .expect("in memory file path is non-empty"),
+    )?;
```

Comment:
Would `to_string_lossy` circumvent the possibility of failing here?

Reply:
09ce2d78b2cf0251d2c21f7396880121fb0286c6